core/services/assets/Registry.php

```diff
@@ -381,7 +379,25 @@ public function getJsUrl($namespace, $chunk_name)
      */
     public function getCssUrl($namespace, $chunk_name)
     {
-        return $this->getAssetUrl($namespace, $chunk_name, self::ASSET_TYPE_CSS);
+        return $this->getAssetUrl($namespace, $chunk_name, Asset::TYPE_CSS);
+    }
+
+
+    /**
+     * @since $VID:$
+     * @throws InvalidArgumentException
+     * @throws InvalidFilePathException
+     */
+    public function registerManifestFiles()
+    {
+        $manifest_files = $this->assets->getManifestFiles();
+        foreach ($manifest_files as $manifest_file) {
+            $this->registerManifestFile(
+                $manifest_file->assetNamespace(),
+                $manifest_file->urlBase(),
+                $manifest_file->filepath() . Registry::FILE_NAME_BUILD_MANIFEST
```

Comment:
I'm not sure this is correct here.  Is manifest name going to have the same name?

Reply:
That's your call I guess. 
Do you want **every** manifest file to have the same name?
They will have different file paths of course, but using the same file name would certainly make it easy to identify them.

You tell me how you want this done.

Reply:
Well that's my point, the third argument is:

```php
$manifest_file->filepath() . Registry::FILE_NAME_BUILD_MANIFEST
```

Unless I'm misunderstanding something, the only unique thing for this argument as you loop through the `$manifest_files` collection is the `filepath()`.  So I was asking you if you were intending for every file manifest file to have the same name because you currently have this using `Registry::FILE_NAME_BUILD_MANIFEST` for every filename.  I think it'd be preferable to just have the third argument be whatever `filepath` returns?

Reply:
> the only unique thing for this argument as you loop through the $manifest_files collection is the filepath()

that's right, and since it appears to me that we only require one manifest file per domain, then even though the filenames would be the same, we would end up with something like:

```
\event-espresso-core\assets\dist\build-manifest.json
\eea-automated-upcoming-event\assets\dist\build-manifest.json
\eea-wait-lists\assets\dist\build-manifest.json
```

which still points to uniques files.
 
So maybe a better question is: 

Is there any reason not to have **all** manifest files named the same like that?

ie: is there any benefit to having something like:

```
\event-espresso-core\assets\dist\build-manifest.json
\eea-automated-upcoming-event\assets\dist\auen.json
\eea-wait-lists\assets\dist\what-exactly-is-this-file.json
```

Is having random file names like that going to make it more difficult to work with them?
I'm guessing it won't make a difference since we just register the full file path and webpack takes care of the rest, but I just want to be absolutely certain.

I personally see no reason to not have all of our manifest files named the same because doing otherwise, means that we now have to have add-ons manually specify the file name they are using, which then also means more boilerplate in this new code I've been adding. Using the same filename also means that an example/empty manifest can be included in the EE add-on skeleton code for everyone to use as a base to start from.

Reply:
Ya what you say makes sense.  It's likely we're just going to be copying the same config between core and add-ons so the filename can just stay the same for add-ons.